doc/docs/master/deploy-manage/deploy/loki.md

```diff
@@ -74,7 +74,15 @@ loki-stack:
        
          ![Container logs](../images/grafana_user_logs.png)
 
-
+## Using Loki in Another Namespace
+Instead of deploying a local loki instance in your pachyderm namespace, you can configure pachyderm to use a loki running in another namespace. To do so, you must
+set `lokiHost` and `lokiPort`. You should also set `lokiDeploy: false` to prevent the chart from deploying a local loki instance.:
+```yaml
+pachd:
+  lokiDeploy: false
+  lokiHost: "<loki-namespace>.<loki-service-name>.svc.cluster.local"
```

Comment:
oh btw, I like to have all k8s DNS addresses in documentation end with `.`, like `loki.monitoring.svc.cluster.local.`.  Do a quick search for "ndots" at k8s.af.  (TL;DR: ending the domain name with a period shows that the name is in the root namespace.  without it, the dns search path is consulted, resulting in extra queries that can overwhelm the DNS service in large clusters.  will this be a problem for our use of loki?  never.  but something to keep in mind, given the number of public postmortems involving ndots problems.)

Reply:
Sure, I can update the documentation to reflect that